src/timer.h

```diff
@@ -1,49 +1,54 @@
 /*
  * This file is
  * Copyright (c) 2009 by Jim Fougeron jfoug AT cox dot net
- * Copyright (c) 2019 by magnum
+ * Copyright (c) 2019-2020 by magnum
  *
  * Redistribution and use in source and binary forms, with or without
  * modification, are permitted.
- *
- * Portable hi-res timer.  Was a nice C++ class. Downgraded to C for john.
  */
 
 #ifndef _HAVE_TIMER_H
 #define _HAVE_TIMER_H
 
-#include <time.h>
-
 /* Windows - use provided hi-res timer */
 #if defined (_MSC_VER) || defined (__MINGW32__) || defined (__CYGWIN32__)
 
-#include <sys/timeb.h>      /* for ftime(), which is not used */
 #undef MEM_FREE
 #include <windows.h>
 #undef MEM_FREE
```

Comment:
I wonder what's the deal with `#undef MEM_FREE` here, but it's better not to touch it for now.

Reply:
This bugged me too. In the end it means the order of `#include` will matter. Looked at push/pop of macros but saw nothing portable and it's not likely anyone but me will include these files right now anyway 🤣